docs/script.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const localBase = './water.css/'
+
+const fileSizes = {
```

Comment:
I think there was a note to double check these before launch, just putting this here in case it hasn't been done yet

Reply:
@kognise double check file sizes?

Reply:
Do we want to wait for #203?

Reply:
#203 and #210 are closed, these should be the release filesizes now

Reply:
Recalculating! Anything else we need to resolve?

Reply:
The only other thing was choosing what the default theme would be for `water.css`, but we can document that in #209.